### ISSUE TITLE
luci-theme-material:Switch port status is hidden after using this theme.

### ISSUE THREAD

**Issue by 981213** [openwrt/luci#784]
I think it might be caused by the following function defined at line 202 in htdocs/luci-static/material/js/script.js:
`

```
$(".cbi-section-table-titles, .cbi-section-table-descr, .cbi-section-descr").each(function () {
    var that = $(this);
    if (that.text().trim() == ""){
        that.css("display", "none");
    }
});
```

`
but I knew little about JavaScript so I don't know how to fix the problem :-(

@LuttyYang I guess this function will hide cbi-section-table-descr if there is no plain text?
But the port status is defined like this:
`

&lt;tr class=&quot;cbi-section-table-descr&quot;&gt;
&lt;th class=&quot;cbi-section-table-cell&quot;&gt;&lt;div id='portstatus-switch0'&gt;&lt;/div&gt;&lt;/th&gt;
&lt;th class=&quot;cbi-section-table-cell&quot;&gt;&lt;div id=&quot;portstatus-switch0-6&quot;&gt;&lt;/div&gt;&lt;/th&gt;
&lt;th class=&quot;cbi-section-table-cell&quot;&gt;&lt;div id=&quot;portstatus-switch0-7&quot;&gt;&lt;/div&gt;&lt;/th&gt;
&lt;th class=&quot;cbi-section-table-cell&quot;&gt;&lt;div id=&quot;portstatus-switch0-0&quot;&gt;&lt;/div&gt;&lt;/th&gt;
&lt;th class=&quot;cbi-section-table-cell&quot;&gt;&lt;div id=&quot;portstatus-switch0-1&quot;&gt;&lt;/div&gt;&lt;/th&gt;
&lt;th class=&quot;cbi-section-table-cell&quot;&gt;&lt;div id=&quot;portstatus-switch0-2&quot;&gt;&lt;/div&gt;&lt;/th&gt;
&lt;th class=&quot;cbi-section-table-cell&quot;&gt;&lt;div id=&quot;portstatus-switch0-3&quot;&gt;&lt;/div&gt;&lt;/th&gt;
&lt;th class=&quot;cbi-section-table-cell&quot;&gt;&lt;div id=&quot;portstatus-switch0-5&quot;&gt;&lt;/div&gt;&lt;/th&gt;
&lt;th class=&quot;cbi-section-table-cell&quot;&gt;&lt;div id=&quot;portstatus-switch0-4&quot;&gt;&lt;/div&gt;&lt;/th&gt;
&lt;/tr&gt;

`
and it will be hidden.

**Comment by finasfinas** [comment]
I can confirm this on a TPLink C5 with recent trunk

**Comment by LGA1150** [comment]
Confirmed in r7312
Can be fixed by removing the `.cbi-section-table-descr, ` in that line.

**Comment by jow-** [comment]
Fixed on latest LuCI master (not 18.06):

![switch](https://user-images.githubusercontent.com/2528802/41936138-805e6e22-798c-11e8-81de-4a3a901f01d3.png)
